packages/react-server/src/test/utilities.ts

```diff
@@ -1,6 +1,37 @@
+import {Server} from 'http';
 import {Context} from 'koa';
+import getPort from 'get-port';
+
 import {KoaNextFunction} from '../types';
 
+export class TestRack {
```

Comment:
We could actually use this to do e2e style tests for all our node server related libraries. We don't need to pull it out into a package now necessarily but it's worth keeping in mind.